### ISSUE TITLE
Please implement MenuFlyoutItem.Click Event

### ISSUE THREAD

**Issue by Daoting** [unoplatform/uno#227]
## I'm submitting a...

  
 - Feature request 

## Current behavior

Please implement `MenuFlyoutItem.Click` Event. Now I can use the `Command `to replace.

## Environment

```
Nuget Package: 

Package Version(s): 

Affected platform(s):
- [x] iOS
- [x] Android
- [x] WebAssembly
- [ ] Windows
- [ ] Build tasks

Visual Studio
- [x] 2017 (version: )
- [ ] 2017 Preview (version: )
- [ ] for Mac (version: )

Relevant plugins
- [ ] Resharper (version: )
```

**Comment by pkar70** [comment]
Yes, please add this event...

**Comment by jeromelaban** [comment]
This was implemented in Uno 3.0

**Comment by ZeProgFactory** [comment]
I don't have the  time to make a sample project, but for me the click event doesn't work in WASM, whereas the commands work fine.

      <PackageReference Include="Uno.UI.RemoteControl" Version="3.0.17" Condition="'$(Configuration)'=='Debug'" />
      <PackageReference Include="Uno.UI.WebAssembly" Version="3.0.17" />
      <PackageReference Include="Uno.Wasm.Bootstrap" Version="1.4.0-dev.16" />
      <PackageReference Include="Uno.Wasm.Bootstrap.DevServer" Version="1.4.0-dev.16" />
